content/posts/newsletter/2019-02-28.md

```diff
@@ -0,0 +1,95 @@
+---
+title: The XMPP Newsletter, 28 February 2019
+save_as: 2019/02/newsletter-28-february/index.html
+url: 2019/02/newsletter-28-february/
+date: 2019-02-28 00:00
+author: jcbrand
+blog_id: newsletter
+category: Newsletter
+---
+
+Welcome to the XMPP newsletter.
+
+If you have an article, tutorial or blog post you'd like us to include in the
+newsletter, please [submit it on the XMPP wiki](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+## News
+
+### FOSDEM
+
+In the beginning of February was the annual [FOSDEM19](https://fosdem.org/2019/) conference in Brussels
+which was well attended by XMPP enthusiasts and where 5 different talks related
+to XMPP were help.
+
+The videos for the FOSDEM presentations have been released:
+
+* [XMPP Beyond Instant Messaging](https://fosdem.org/2019/schedule/event/xmpp_beyond_im/)
+
+In this talk Jérôme Poisson (Goffi) shows how XMPP can be used to build non-IM
+applications, such as a blog engine, file sharing, a code forge, a universal remote
+controller and even a decentralized web framework on top of XMPP.
+
+* [Salut à Toi: A Python Based Social Network And More](https://fosdem.org/2019/schedule/event/python_salut_a_toi_network/)
+
+Jérôme Poisson talks about Salut à Toi, a Python-based decentralized communication ecosystem
+which uses XMPP and offers a lot of features like chat, e2e encryption, file sharing, photo
+albums, events, (micro)blogging, forums, remote control, tickets,
+merge-requests and even a decentralized web framework.
+
+* [Make XMPP Sprint Again](https://fosdem.org/2019/schedule/event/xmpp_sprint/)
+
+Maxime Buquet (pep.) talks about how he organized the August 18-19th XMPP
+community sprint in Cambridge and gives useful advice to anyone interesting in
+organising community events such as development sprints.
+
+* [Unified Communications with Pàdé](https://fosdem.org/2019/schedule/event/pade/)
+
+Dele Olajide talks about Pàdé, a new unified communications solution from the
+ignite Real-time community that integrates SIP, HTTP and XMPP into one desktop
+solution.
```

Comment:
Can we focus more on the use cases rather than the protocol? That would be something "real time communications, including text, voice and video"?

Reply:
You're welcome to suggest different wording, but please also include the protocols because that's useful to know for some people.

Reply:
ok

Reply:
"that integrates protocols such as SIP, HTTP and XMPP into one desktop solution for 
text, voice and video communications": is it better?